kr/components/card.md

```diff
@@ -6,7 +6,10 @@ _language: kr
 ---
 
 ##Card
```

Comment:
There should be a space after the `#` for headings to work. 
![image](https://github.com/IgniteUI/igniteui-docfx/assets/19392175/e4edc4a7-7b2f-4aba-8ea5-9efbaa870426)